components/ModDashboardHealthProfessionalCard.vue

```diff
@@ -55,15 +55,15 @@
                 </div>
             </div>
             <div
-                v-if="healthcareProfessionalNameByLocale || removedHealthcareProfessionalName"
```

Comment:
Were these unused variables introduced by this PR? 🤔 If so, we should avoid creating and removing something in the same PR.